dask/dataframe/tests/test_groupby.py

```diff
@@ -2187,3 +2187,64 @@ def test_groupby_transform_ufunc_partitioning(npartitions, indexed):
                 lambda series: series - series.mean()
             ),
         )
+
+
+@pytest.mark.parametrize("dropna", [False, True, None])
+def test_groupby_dropna_pandas(dropna):
```

Comment:
I'd prefer that this test implements the behavior we expect pandas to adopt, and be marked with an `xfail`.

Then, when pandas does add support for `dropna` this test will start failing and we'll just have to remove the xfail (for new pandas).

So, can you remove the `dropna=None` case and change the TypeErrors to the expected result?

Reply:
Good suggestion - Using xfail now.